pupil_src/shared_modules/video_export/plugins/eye_video_exporter.py

```diff
@@ -93,7 +93,7 @@ def __call__(self, _, frame):
             pupil_datum = self._pupil_positions_of_eye.by_ts(frame.timestamp)
             draw_pupil_on_image(eye_image, pupil_datum)
         except ValueError:
-            if not self._warned_once:
-                logger.warning("Inconsistent timestamps found in pupil data")
-                self._warned_once = True
+            if not self._warned_once_data_not_found:
+                logger.warning("Pupil data for visualization not found")
```

Comment:
Maybe `"Could not draw pupil position, because there is no pupil position in frame."`